### PR TITLE
feat(pingcap/tidb): support more integration pipeline in VerfiyCI of tidb lts branches

### LINKED ISSUE #3057: [TiDB@release-6.5] Support more integration pipeline in VerfiyCI of tidb lts branches

### LINKED ISSUE #3059: [TiDB@release-6.1] Support more integration pipeline in VerfiyCI of tidb lts branches

### PR DESCRIPTION
**Pull request by wuhuizuo**
- release-6.1
- release-6.5

Fixes #3057
Fixes #3059

Signed-off-by: wuhuizuo <wuhuizuo@126.com>

### CODE REVIEW

File: jobs/pingcap/tidb/release-6.1/pull_integration_br_test.groovy
Comment (purelind) [inline jobs/pingcap/tidb/release-6.1/pull_integration_br_test.groovy:1]: There is already a pull_br_integration_test, no need to add a new one here.

File: jobs/pingcap/tidb/release-6.1/pull_integration_lightning_test.groovy
Comment (purelind) [inline jobs/pingcap/tidb/release-6.1/pull_integration_lightning_test.groovy:1]: This branch's lightning pipeline has not been split, so this pipeline is not needed.

File: jobs/pingcap/tidb/release-6.5/pull_integration_br_test.groovy
Comment (purelind) [inline jobs/pingcap/tidb/release-6.5/pull_integration_br_test.groovy:1]: please delete this，pull_br_ingration_test alreay exist

File: jobs/pingcap/tidb/release-6.5/pull_integration_lightning_test.groovy
Comment (purelind) [inline jobs/pingcap/tidb/release-6.5/pull_integration_lightning_test.groovy:1]: This branch's lightning pipeline has not been split, so this pipeline is not needed, please delete this

File: pipelines/pingcap/tidb/release-6.1/pod-pull_e2e_test.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+          memory: 16Gi
+          cpu: "6"
+    - name: net-tool
+      image: hub.pingcap.net/jenkins/network-multitool
+      tty: true
+      resources:
+        limits:
+          memory: 128Mi
+          cpu: 100m
+    - name: report
```
Comment (purelind) [inline pipelines/pingcap/tidb/release-6.1/pod-pull_e2e_test.yaml:24]: please remove this unused container `report`

File: pipelines/pingcap/tidb/release-6.1/pod-pull_integration_copr_test.yaml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+          memory: 16Gi
+          cpu: "20"
+    - name: net-tool
+      image: hub.pingcap.net/jenkins/network-multitool
+      tty: true
+      resources:
+        limits:
+          memory: 128Mi
+          cpu: 100m
+    - name: report
```
Comment (purelind) [inline pipelines/pingcap/tidb/release-6.1/pod-pull_integration_copr_test.yaml:21]: please remove this unused container report

File: pipelines/pingcap/tidb/release-6.1/pod-pull_common_test.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
+apiVersion: v1
+kind: Pod
+spec:
+  securityContext:
+    fsGroup: 1000
+  containers:
+    - name: golang
+      image: "hub.pingcap.net/jenkins/centos7_golang-1.21:latest"
```
Comment (purelind) [inline pipelines/pingcap/tidb/release-6.1/pod-pull_common_test.yaml:8]: go version is go1.20，please update to `hub.pingcap.net/jenkins/centos7_golang-1.20:latest`

File: pipelines/pingcap/tidb/release-6.1/pod-pull_e2e_test.yaml [hunk 4]
```diff
@@ -0,0 +1,39 @@
+apiVersion: v1
+kind: Pod
+spec:
+  securityContext:
+    fsGroup: 1000
+  containers:
+    - name: golang
+      image: "hub.pingcap.net/jenkins/centos7_golang-1.21:latest"
```
Comment (purelind) [inline pipelines/pingcap/tidb/release-6.1/pod-pull_e2e_test.yaml:8]: go version is go1.20，please update to hub.pingcap.net/jenkins/centos7_golang-1.20:latest

File: pipelines/pingcap/tidb/release-6.5/pod-pull_common_test.yaml [hunk 5]
```diff
@@ -0,0 +1,42 @@
+apiVersion: v1
+kind: Pod
+spec:
+  securityContext:
+    fsGroup: 1000
+  containers:
+    - name: golang
+      image: "hub.pingcap.net/jenkins/centos7_golang-1.21:latest"
```
Comment (purelind) [inline pipelines/pingcap/tidb/release-6.5/pod-pull_common_test.yaml:8]: go version is go1.20，please update to hub.pingcap.net/jenkins/centos7_golang-1.20:latest

**Review by purelind — CHANGES_REQUESTED** [review]

File: prow-jobs/pingcap/tidb/release-6.5-presubmits.yaml [hunk 6]
```diff
@@ -56,5 +53,103 @@ presubmits:
...
+    - name: pingcap/tidb/release-6.5/pull_lightning_integration_test
+      agent: jenkins
+      decorate: false # need add this.
+      context: pull-lightning-integration-test
+      always_run: false
+      optional: true
+      skip_report: false
+      trigger: "(?m)^/test (?:.*? )?pull-lightning-integration-test(?: .*?)?$"
+      rerun_command: "/test pull-lightning-integration-test"
+      branches: *branches
```
Comment (purelind) [inline prow-jobs/pingcap/tidb/release-6.5-presubmits.yaml:66]: Please remove those lines. Not exist lighting pipelines in 6.5(br & lightning are in the same dir)

File: prow-jobs/pingcap/tidb/release-6.1-presubmits.yaml [hunk 7]
```diff
@@ -56,5 +53,103 @@ presubmits:
...
+    - name: pingcap/tidb/release-6.1/pull_lightning_integration_test
+      agent: jenkins
+      decorate: false # need add this.
+      context: pull-lightning-integration-test
+      always_run: false
+      optional: true
+      skip_report: false
+      trigger: "(?m)^/test (?:.*? )?pull-lightning-integration-test(?: .*?)?$"
+      rerun_command: "/test pull-lightning-integration-test"
+      branches: *branches
```
Comment (purelind) [inline prow-jobs/pingcap/tidb/release-6.1-presubmits.yaml:66]: Please remove those lines. This branch's lightning pipeline has not been split from br, so this pipeline is not needed.

**Review by purelind — CHANGES_REQUESTED** [review]

File: prow-jobs/pingcap/tidb/release-6.1-presubmits.yaml (hunk 7 above)
Comment (wuhuizuo) [inline prow-jobs/pingcap/tidb/release-6.1-presubmits.yaml:66]:
```suggestion
```

File: prow-jobs/pingcap/tidb/release-6.5-presubmits.yaml (hunk 6 above)
Comment (wuhuizuo) [inline prow-jobs/pingcap/tidb/release-6.5-presubmits.yaml:66]:
```suggestion
```

**Review by purelind — APPROVED** [review]
/lgtm

**Pull request merged**